packages/xo-web/src/xo-app/vm-import/ova/index.js

```diff
@@ -1,3 +1,5 @@
+/* eslint no-console: "off" */
+
```

Comment:
This change is unrelated to this PR, it would be best to avoid it.

Why did you need to do this?

ESLint it automatically ran on changed files, which is not the case for this one, therefore you should not have seen this warning.

Reply:
I don't know, I got an error with a blocked commit, I did what I could to get the commit through, I have been trying to get the PR through for 2 years, I'm just rolling with the punches here.

Reply:
Do you want me to rebase your branch on top of master, squashing all commits and reverting trivial issues?

Reply:
yes please.